src/netstandard/ref/Directory.Build.props

```diff
@@ -4,6 +4,7 @@
     <!-- We need to move to 1.1 as part of our versioning change when we switch to arcade to avoid downgrading versions -->
     <MajorVersion>1</MajorVersion>
     <MinorVersion>1</MinorVersion>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
```

Comment:
Why'd you need to change this?

Reply:
As we only build this for netstandard and the project's name is the same it doesn't make sense to append the tfm. This also guarantees to keep what we had before as I removed the property in the root script: https://github.com/dotnet/standard/pull/1040/files#diff-4b2f402501d23abbede6eac0f47783e4L33